docs/diagrams/ClearOverdueEntryCommandActivityDiagram.puml

```diff
@@ -0,0 +1,14 @@
+@startuml
+
+start
+
+repeat
+  :Model checks entry;
+    if (Is the end date of entry passed?) then (yes)
```

Comment:
nope. the diamond may not have any text in it. the statements should be on the branches that extend out of the diamond